doc/release/1.12.0-notes.rst

```diff
@@ -23,6 +23,20 @@ Future Changes
 Compatibility notes
 ===================
 
+DeprecationWarning to error
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Indexing with floats raises IndexError,
+  e.g., a[0, 0.0].
```

Comment:
Double ticks missing. Anyway, looks all good to me otherwise, only those nitpicks.